packages/react-ui/internal/Popup/Popup.tsx

```diff
@@ -551,6 +555,10 @@ export class Popup extends React.Component<PopupProps, PopupState> {
     }
 
     const location = this.getLocation(popupElement, this.state.location);
+    if (this.props.position) {
+      this.props?.position(location?.position as PopupPositionsType);
+    }
```

Comment:
Тут небольшая чехарда с проверками на наличие. К тому же, получается, что допускается вызов коллбэка с аргументами `null`/`undefined`, хотя по типам "выше" везде ожидается строго `PopupPositionsType`.

Предлагаю такой вариант, чтобы сразу всё поправить:
```suggestion
    if (location) {
      this.props.position?.(location.position);
    }
```

Reply:
Поправил